actions/admin/site/flush_cache.php

```diff
@@ -3,8 +3,7 @@
  * Flush all the caches
  */
 
-elgg_invalidate_simplecache();
-elgg_reset_system_cache();
+elgg_flush_caches();
 _elgg_services()->autoloadManager->deleteCache();
```

Comment:
I tried to find a correct place to do this, but couldn't find one i was happy with. Suggestions?